.conf/dps_27/nginx.tasmoadmin.conf

```
@@ -1,6 +1,8 @@
 location ^~ /tasmoadmin {
-	location /tasmoadmin/firmwares {
-	}
+	# Allow access to firmwares
+	location ~ ^/tasmoadmin/data/firmwares {
+                allow all;
+        }
```

This should work as well
```suggestion
	location ~ ^/tasmoadmin/data/firmwares {
        }
```